tests/FSharp.Editing.Tests/GoToDefinitionTests.fs

```diff
@@ -260,7 +260,7 @@ type Console =
     static member Beep : unit -> unit
 """
 
-[<Test>]
+[<Test; Ignore "Need extra xml files on Travis">]
```

Comment:
I think those tests should be in a separate category and should be ignored on Travis.